{
  try {
    nimm.testInvoke();
  }
 catch (  Exception e) {
    e.printStackTrace();
    return e.getMessage();
  }
  return "OK";
}
